spec_analysis/jw8_redshift_fitting_solutions.ipynb

```diff
@@ -25,7 +25,9 @@
     "    - Get the library of spectra\n",
```

Comment:
<p>The "Installation" section in the notebooks suggests only installing the latest release of jdaviz, but there's also a requirements.txt with pinned versions of multiple packages. If relying on the requirements file, then the version printing might not be necessary to include in the notebook.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/spacetelescope/jwebbinar_prep/pull/89/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='2'/>

Reply:
Making it clearer that one can either install all the packages needed to run all the notebooks in JWebbinar 8 or just jdaviz for this specific notebook.